README.md

```diff
@@ -133,6 +141,7 @@ Michael Koval ([**@mkoval**](https://github.com/mkoval))
 and Pras Velagapudi ([**@psigen**](https://github.com/psigen)).
 It has received major contributions from
 Shushman Choudhury ([**@Shushman**](https://github.com/Shushman)),
+Ethan Gordon ([**@egordon**](https://github.com/egordon)),
```

Comment:
👏

Reply:
Hi! :)

Felt comfortable with that at this point. Feel free to add yourself once you have a few hundred merged lines under your belt.